Plugins/NativeCamera/Source/Apple/NativeCameraImpl.mm

```diff
@@ -347,7 +350,9 @@ fragment float4 fragmentShader(RasterizerData in [[stage_in]],
             // Actually start the camera session.
             [m_implData->avCaptureSession addOutput:dataOutput];
             [m_implData->avCaptureSession commitConfiguration];
-            [m_implData->avCaptureSession startRunning];
+            dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
```

Comment:
Does `dispatch_get_global_queue` have a specific thread affinity? I took a quick look at the docs and couldn't tell. Overall what is this `dispatch_async` call trying to do?

Reply:
The goal is to run this on any background thread, as running it on the main thread will hold up app responsiveness.  In general the important one to offload is stopRunning as that will lock up the app for ~3 seconds if run on the main thread, but a warning is generated when running this on the main thread as well.